metaflow/client/core.py

```diff
@@ -376,7 +344,8 @@ def __init__(self,
         else:
             raise MetaflowInternalError(msg="Unknown type: %s" % self._NAME)
 
-        self._created_at = datetime.fromtimestamp(self._object['ts_epoch']/1000.0)
+        self._created_at = time.strftime(
```

Comment:
I would revert this because when we merged #692 in master and then back to here, it will conflict needlessly.

Reply:
How will this conflict? It's a no-change from master and should flow seamlessly on rebase.